ONSdigital/census-rh-ui
# Cr 1733 missing attributes

# Motivation and Context
Updates all calls for session data (both explicit and implicit) to check the data exists in the session first

# What has changed
Updates calls for session data to check for the data before trying to use it
Change to some page 'return' calls to make the explicit as to the data sent to the page, to ensure all needed values are checked to exist
Addition of a session forget call in start of fulfilment journey to prevent cross session issues

No Cucumber changes required

Note addresses issues in multiple Jira tickets and incidents

# How to test?
Try re-creating the conditions that triggered the various errors. Ensure issue is resolved.

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1733
https://collaborate2.ons.gov.uk/jira/browse/CR-1744
https://collaborate2.ons.gov.uk/jira/browse/CR-1734
https://collaborate2.ons.gov.uk/jira/browse/CR-1755
